[2022-09-08 11:05:12,065] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example3.t1 scheduled__2021-01-01T00:00:00+00:00 [queued]>
[2022-09-08 11:05:12,071] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example3.t1 scheduled__2021-01-01T00:00:00+00:00 [queued]>
[2022-09-08 11:05:12,071] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-08 11:05:12,071] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2022-09-08 11:05:12,071] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-08 11:05:12,084] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2021-01-01 00:00:00+00:00
[2022-09-08 11:05:12,086] {standard_task_runner.py:52} INFO - Started process 11711 to run task
[2022-09-08 11:05:12,091] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example3', 't1', 'scheduled__2021-01-01T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/example3.py', '--cfg-path', '/var/folders/ll/lb_b4vvx7cgfp2wpfsv4w6zm0000gn/T/tmpfoj2y1tz', '--error-file', '/var/folders/ll/lb_b4vvx7cgfp2wpfsv4w6zm0000gn/T/tmpmk31ubt6']
[2022-09-08 11:05:12,092] {standard_task_runner.py:80} INFO - Job 22: Subtask t1
[2022-09-08 11:05:12,140] {task_command.py:371} INFO - Running <TaskInstance: example3.t1 scheduled__2021-01-01T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-09-08 11:05:12,194] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example3
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-01T00:00:00+00:00
[2022-09-08 11:05:12,234] {local_task_job.py:156} INFO - Task exited with return code -11
[2022-09-08 11:05:12,254] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=example3, task_id=t1, execution_date=20210101T000000, start_date=20220908T030512, end_date=20220908T030512
[2022-09-08 11:05:12,286] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
